Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -858,6 +860,7 @@ namespace FluentAssertions.Equivalency
     {
         System.Type DeclaringType { get; }
         FluentAssertions.Common.CSharpAccessModifier GetterAccessibility { get; }
+        bool IsBrowsable { get; }
```

Comment:
Just for the record, as adding a member to a public interface is a source breaking change.
In https://github.com/fluentassertions/fluentassertions/pull/1742#discussion_r790274160 we agreed changes to `INode` are justified as we believe(/hope) that they are not implemented by anyone else.
I'm fine with applying the same argument to changes to `IMember`.